<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPreferences</class>
 <widget class="QDialog" name="frmPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/notepadqq/scalable/notepadqq.svg</normaloff>:/icons/notepadqq/scalable/notepadqq.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Appearance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Languages</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Search</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shortcuts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toolbar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extensions</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageGeneral">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkCheckQtVersionAtStartup">
           <property name="text">
            <string>Check Qt version at startup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkCollectStatistics">
           <property name="text">
            <string>Collect and transmit anonymous statistics to improve Notepadqq</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkWarnForDifferentIndentation">
           <property name="text">
            <string>Warn when the indentation doesn't match the settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkRememberSession">
           <property name="text">
            <string>Remember open tabs when closing Notepadqq</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkExitOnLastTabClose">
           <property name="text">
            <string>Exit Notepadqq when closing the last Tab</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="localizationLabel">
             <property name="text">
              <string>Localization:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="localizationComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageAppearance">
        <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0,1">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Color scheme:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbColorScheme"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="chkOverrideFontFamily">
             <property name="text">
              <string>Override Font Family</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFontComboBox" name="cmbFontFamilies">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="chkOverrideFontSize">
             <property name="text">
              <string>Override Font Size</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spnFontSize">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>28</number>
             </property>
             <property name="value">
              <number>14</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="chkOverrideLineHeight">
             <property name="text">
              <string>Override Line Height</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="spnLineHeight">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string>em</string>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.300000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="colorSchemePreviewFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageLanguages">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmbLanguages"/>
         </item>
         <item>
          <widget class="QFrame" name="frameLanguages">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>121</width>
              <height>27</height>
             </rect>
            </property>
            <property name="text">
             <string>Tab size:</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="chkLanguages_IndentWithSpaces">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>361</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>Use spaces instead of tabs</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="txtLanguages_TabSize">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>10</y>
              <width>71</width>
              <height>27</height>
             </rect>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkLanguages_useDefaultSettings">
           <property name="text">
            <string>Use default settings</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageSearch">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkSearch_SearchAsIType">
           <property name="text">
            <string>Search as I type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSearch_SaveHistory">
           <property name="text">
            <string>Save search history</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageShortcuts"/>
       <widget class="QWidget" name="pageToolbar">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="listToolbarAll"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnToolbarAdd">
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset theme="go-next"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnToolbarRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset theme="go-previous"/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnToolbarUp">
                <property name="text">
                 <string>Move Up</string>
                </property>
                <property name="icon">
                 <iconset theme="go-up"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnToolbarDown">
                <property name="text">
                 <string>Move Down</string>
                </property>
                <property name="icon">
                 <iconset theme="go-down"/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QListWidget" name="listToolbarCurrent"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnToolbarReset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Reset to Default</string>
             </property>
             <property name="icon">
              <iconset theme="edit-undo"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageExtensions">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>362</width>
              <height>205</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Node.js runtime</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Supported Node versions: 0.10, 0.11, 0.12</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Node.js path:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="txtNodejs"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnNodejsBrowse">
                    <property name="text">
                     <string>Browse...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>NPM path:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLineEdit" name="txtNpm"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnNpmBrowse">
                    <property name="text">
                     <string>Browse...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>WARNING: support for extensions is EXPERIMENTAL.</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
